symplyphysics/laws/electricity/circuits/power_by_resistor_in_alternating_current_circuit.py

```diff
@@ -0,0 +1,44 @@
+from sympy import (Eq, solve, sin)
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+    validate_output, angle_type)
+
+# Description
+## The resistor is in an alternating current circuit. Then the thermal power dissipated on the resistor will depend on the amplitude of the current,
+## the frequency of the alternating current, the time and the resistance of the resistor.
+
+## Law is: P = I^2 * R * sin(w * t)^2, where
+## P - power,
```

Comment:
There are various types of power to measure, eg instantaneous, average, RMS. Specify, which one is it.